### PR TITLE
INT-331 -  give scheduler extra permissions

### PR DESCRIPTION
**Pull request by Mojachieee**
Thanks @nolancon for figuring out what was wrong

### CODE REVIEW

File: bundle/manifests/storageos_operator_rbac.authorization.k8s.io_v1_clusterrole.yaml [hunk 1]
```diff
@@ -124,9 +124,12 @@ rules:
   - csi.storage.k8s.io
   resources:
   - csidrivers
+  - csistoragecapacities
```
Comment (Mojachieee) [inline bundle/manifests/storageos_operator_rbac.authorization.k8s.io_v1_clusterrole.yaml:127]: I still need to figure out if the changes to this file are necessary

Comment (Mojachieee) [reply bundle/manifests/storageos_operator_rbac.authorization.k8s.io_v1_clusterrole.yaml:127]: Turns out this file is auto-generated. 

Running `make bundle` updates it

File: channels/packages/scheduler/1.18.0/cluster-role.yaml [hunk 2]
```diff
@@ -37,6 +37,8 @@ rules:
   resources:
   - storageclasses
   - csinodes
+  - csistoragecapacities
+  - csidrivers
```
Comment (nolancon) [inline channels/packages/scheduler/1.18.0/cluster-role.yaml:41]: were these changes autogenerated?

Comment (Mojachieee) [reply channels/packages/scheduler/1.18.0/cluster-role.yaml:41]: Nope they're not 

(discussed in DMs)

**Review by nolancon — APPROVED** [review]

**Pull request merged**